prow/cmd/webhook-server/helpers.go

```diff
@@ -18,16 +15,73 @@ package main
 
 import (
 	"bytes"
+	"context"
 	cryptorand "crypto/rand"
 	"crypto/rsa"
 	"crypto/x509"
 	"crypto/x509/pkix"
+	"encoding/json"
 	"encoding/pem"
+	"errors"
 	"fmt"
+	"io/ioutil"
 	"math/big"
+	"os"
+	"path/filepath"
 	"time"
+
+	"github.com/sirupsen/logrus"
+	admregistration "k8s.io/api/admissionregistration/v1"
+	b64 "encoding/base64"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	prowflagutil "k8s.io/test-infra/prow/flagutil"
+	ctrlruntimeclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const fsSecretPath = "./prowjob-webhook-ca-cert"
```

Comment:
I would expect this to be a command line argument option. And the decision would be:
- if project id is provided then store in GCP
- if local file path is provided then store in local file system
- if both are provided then fail